feature/experimental/bgp/ate_tests/base_bgp_session_parameters/README.md

```diff
@@ -12,7 +12,8 @@ BGP session establishment between DUT - ATE and verifiying different session par
 
 Test the abnormal termination of session using notification message:
 
-*   Establish BGP session between DUT (AS 65540) and ATE (AS 65550).
+*   Establish BGP session between DUT (AS 65540) and ATE (AS 65550). The DUT/ATE
+    peers should be configured with matching passwords.
```

Comment:
The DUT/ATE peers should be configured using same MD5 authentication password.

Reply:
Updated the wording to specifically mention MD5.